src/utils/saveTicketTranscript.ts

```
@@ -92,12 +93,29 @@ export default async function (
+    if (message.author.id === client.user?.id) {
+      if (
+        message.content &&
+        message.content === supportConfig.ticketWelcomeMessage
+      )
+        continue;
+
+      if (message.embeds.length > 0) {
+        const embed = message.embeds[0];
+        const [_, __, formTitle] = supportConfig.ticketCategories
+          .find((cat: string) => cat.includes(ticketData.category))
+          ?.split(":");
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)